### DISCUSSION TITLE
EL2 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#218] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/EL2.html

Author: @AbbyANoble 
Reviewer: @kathypinzon

**Comment by kathypinzon** [comment]
Gridlines in graph for 5.2.1 (a) answer are not complete.
5.2.1 (e) I would go with answer choices F-I and not give specific numbers. Maybe you could add another part that says, What are the equations of the asymptotes? and make the answer choices similar to choices A-E

**Comment by kathypinzon** [comment]
Gridlines in graph for 5.2.2 (a) answer are not complete.
5.2.1 (e) Answer has a stray >

**Comment by kathypinzon** [comment]
5.2.6 (d) is missing an answer (Should be D.)

**Comment by kathypinzon** [comment]
5.2.8 - I would add another part to the question asking about what happens to the domain, range, and asymptotes.  This is the first time they are experiencing transformations of graphs with asymptotes, so it would be useful to note.  Maybe one task for each of the 4 transformed graphs where they give the domain, range and asymptote for the transformed function.

**Reply by AbbyANoble** [reply]
Good idea! I updated this one. Let me know what you think!

**Comment by kathypinzon** [comment]
5.2.9 (a) is missing a space after the end of the second sentence before the parentheses. I would include asking them to state the domain, range and asymptote.
5.2.9(b) and (c) needs the transformations applied in the answer. Again, domain, range, asymptote

**Comment by kathypinzon** [comment]
5.2.10 I would ask them to write the equation of the asymptote as well.  include domain and range in answers.

**Comment by AbbyANoble** [comment]
Should I have the asymptotes dotted in on these graphs?  [Activity 5.2.8(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EL2.html#activity-exp-graphs-match-3)

**Reply by kathypinzon** [reply]
yeah, If that's possible it would be a good idea.  Make it easier for them to see and reinforce that when the graph exponentials and logs they should plot the asymptote.
